scripts/import-ai-lab-samples

```diff
@@ -57,6 +61,9 @@ for f in */; do
         sed -i "s!sed.edit.APPTAGS!$APP_TAGS!g" $DEST/template.yaml
         sed -i "s!sed.edit.CATALOG_DESCRIPTION!Secure Supply Chain Example for $APP_DISPLAY_NAME!g" $DEST/template.yaml
 
+        sed -i "s!sed.edit.NAME!$APP_NAME!g" $DEST/docs/index.md
+        sed -i "s!sed.edit.TEMPLATE_URL!$TEMPLATE_SOURCE_URL!g" $DEST/docs/index.md
```

Comment:
```suggestion
        sed -i "s!sed.edit.TEMPLATE_SOURCE_URL!$TEMPLATE_SOURCE_URL!g" $DEST/docs/index.md
```

nit: I would try to keep the same naming, i feel it will add to confusion when/if more ENVs are introduced later

Reply:
Updated the naming as part of https://github.com/redhat-ai-dev/ai-lab-template/pull/48/commits/cdb21c2d5db4c8f77dca633ad568b254f43fb059 and also made some of the prior code consistent with these name changes here https://github.com/redhat-ai-dev/ai-lab-template/pull/48/commits/36c7a01a79202e9fcbf0ce8e9902cf166125eabe